subset/bacnet/Dockerfile.test_bacext

```diff
@@ -18,6 +18,10 @@ RUN $AG update && $AG install openjdk-8-jdk git
 
 COPY subset/bacnet/ .
 
+COPY subset/bacnet/bacnetTests/src/main/resources/csv tmp/
```

Comment:
this is instead of the volume mapping  ?

Reply:
yes, if the correct csv is not found then it will look at this temp csv